api/models/site.js

```diff
@@ -113,6 +114,10 @@ function isEmptyOrUrl(value) {
   }
 }
 
+function isS3BucketDedicated() {
+  return this.s3ServiceName !== `federalist-${config.app.app_env}-s3`;
```

Comment:
This seems a bit brittle (1) and risky (2) to me. If the shared bucket name changes for whatever reason, this will return true and allow Netlify CMS to be used.

1. Is there a more robust way to absolutely ensure that the site is using a dedicated bucket? (I thought there was a configuration value somewhere...)

2. I think we should err on the side of a false negative here and only return true if we are sure that a dedicated bucket is used.

Reply:
agreed ... good feedback ... will adjust

Reply:
adjusted

Reply:
this is 👍 
Feels weird to determine this by the bucketname, not sure why we wouldn't just have an attribute on the Site model that indicates the bucket type....